Currency.py

```diff
@@ -14,6 +15,7 @@
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Currency converter"
 __version__ = "1.0"
+__trigger__ = "cc "
```

Comment:
why do you introduce a trigger? actually it was intended to work without the trigger.